dropwizard-testing/src/main/java/io/dropwizard/testing/junit5/DAOTestExtension.java

```diff
@@ -70,6 +74,20 @@ public void after() {
         daoTest.after();
     }
 
+    @Override
+    public void beforeAll(ExtensionContext extensionContext) throws Exception {
+        try {
+            daoTest.before();
```

Comment:
Would it make sense to call `DAOTestExtension#before()` here so that the behavior is guaranteed to be the same?